IssueTracker/Models/Request/FileRequest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.AspNetCore.Http;
+
+namespace Models.Request;
+public class FileRequest
+{
+    public FileRequest(IFormFile formFile, int? issueId, int? commentId)
```

Comment:
Optional properties should not be present in a constructor.

Reply:
ok